lib/elixir/lib/uri.ex

```diff
@@ -76,7 +76,7 @@ defmodule URI do
   values are URL encoded as per `encode_www_form/1`.
 
   Keys and values can be any term that implements the `String.Chars`
-  protocol, except lists which are explicitly forbidden.
+  protocol, except lists – which are explicitly forbidden.
```

Comment:
Do you have resources on why this is a good change? Not a native speaker but it feels like this was fine before?

Reply:
Note that we generally avoid such constructs, exactly because this increases the barrier for writing the docs. Should we use `-` or should we not? Same for `;` vs `.`/`,`.

Reply:
> Keys and values can be any term that implements the `String.Chars` protocol, except lists which are explicitly forbidden.

The original sentence may be understood as 
> Keys and values can be any term that implements the `String.Chars` protocol, except lists **that** are explicitly forbidden.

Which reads as some lists in particular are forbidden

I think we need a pause, to strees that any list is forbidden
My take is

> Keys and values can be any term that implements the `String.Chars` protocol with the exception of lists, which are explicitly forbidden.

or

> Keys and values can be any term that implements the `String.Chars` protocol with the exception of lists (which are explicitly forbidden).

Reply:
Let's go with

> Keys and values can be any term that implements the `String.Chars` protocol with the exception of lists, which are explicitly forbidden.

Reply:
done